app/api/entries/[id]/route.js

```diff
@@ -0,0 +1,40 @@
+import { dbConnect } from '@/utils/dbConnects';
+import Entry from '../models/Entry';
+import { getUserFromCookie } from '@/utils/auth';
+export async function DELETE(req) {
```

Comment:
nitpick;

Reply:
What is it that you'd like to have changed? 😄

Reply:
New line between the imports and the function :). 
My suggestion was gone with the code change